downloader.py

```diff
@@ -25,6 +25,7 @@
 levels = [1, 2, 3, 5, 7, 20, 30, 70, 150, 350, 450, 550, 600, 650, 750, 800, 900, 950, 975]
 NUM_MEMBERS = 20
 MAX_HOURS = 384
+INTERVAL = 21600
```

Comment:
Since this is global, please name it something more descriptive like FORECAST_INTERVAL. Also please use the value 6 (and replace all occurrences of the value 6 with the constant) since we mostly want to deal in hours, except for the windfile itself where we have to store the unix timestamp which is in seconds.